tests/e2e/test_launch.py

```
@@ -0,0 +1,111 @@
+            cwd=self.project_root,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+        )
+
+        try:
+            time.sleep(5)
+
+            if process.poll() is not None:
+                stdout, stderr = process.communicate()
+                assert False, f"Streamlit crashed:\n{stderr}"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)